src/components/concept/_notification-deduplication.mdx

```diff
@@ -0,0 +1,8 @@
+## Notification and alert de-duplication
+
+Notifications are sent only the first time a given finding is detected.
```

Comment:
Minor suggestion: Add a link to Findings in CI (https://semgrep.dev/docs/semgrep-ci/findings-ci) so that users know what makes a finding "unique"?